app/views/inputBox.view.js

```diff
@@ -7,24 +6,37 @@ var MainView = Backbone.View.extend({
         this.render(options);
     },
     render: function () {
-        var someHtml = _.templateFromUrl("./app/templates/inputBox.template.html");
+        var inputHtml = _.templateFromUrl("./app/templates/inputBox.template.html");
         var canvasHtml = _.templateFromUrl("./app/templates/canvas.template.html");
         this.$el.find("#canvas").html(canvasHtml());
-console.log(canvasHtml());
-        this.$el.find('#input1').html(someHtml({ind: 1}));
-        this.$el.find('#input2').html(someHtml({ind: 2}));
+        this.$el.find('#input1').html(inputHtml({ind: 1}));
+        this.$el.find('#input2').html(inputHtml({ind: 2}));
     },
     events: {
         //'change input': 'drawRectangle'
-        'change input': 'getValue'
+        'keyup input': 'getValue'
     },
-    getValue: function (evt) {
-        var target = $(evt.currentTarget);
-        data = {};
-        data[target.attr('data-coord')] = parseInt(target.val(), 10);
+    getValue: function () {
+        var target = $(':focus');
+        console.log(target);
+        var value = parseInt(target.val(), 10);
+        var data = {};
+        data[target.attr('data-coord')] = value;
         var index = parseInt(target.attr('index'), 10);
+        var rectangle = this.collection.at(index - 1);
+        rectangle.set(data);

```

Comment:
to push backbone to validate data you have to add `validate` option. Check it here http://backbonejs.org/#Model-validate